examples/table/LICENSE

```diff
@@ -0,0 +1,28 @@
+
+Copyright (c) 2017, Jonas Schell
```

Comment:
Another case of "is this file required?". Since it's being added to an existing repository, we don't need sublicenses (unless you particularly wish to explicitly retain copyright to this piece of code)